colorscad.sh

```diff
@@ -265,6 +265,8 @@ done
 # Wait for all remaining jobs to finish
 wait
 
+COLORS=$(echo "$COLORS" | sort -n)
```

Comment:
Let's add some comment. Also the `-n` has no effect I think (lines start with a `[` anyway) so it could be omitted.
```suggestion
# Now sort colors by value, to reduce the need for remapping slicer colors when iteratively designing
COLORS=$(echo "$COLORS" | sort)
```

Reply:
yep, you are right about the -n
my thought process was to have colors ordered by rgb values and not have the wired 
1
10
11
...
2
20

ordering, but that's not how that works :-)